pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -53,6 +54,9 @@ const (
 	DefaultCNIDir     = "/opt/cni/bin"
 
 	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
+	// private mac prefix safe to use
```

Comment:
The second-least-significant bit of the first octet of the address. 1 stands for locally administered address